frontend/src/components/UpdateProjectForm.tsx

```diff
@@ -71,12 +69,13 @@ export default function UpdateProjectForm(props: UpdateProjectProps) {
                 autoComplete="off"
             >
                 <div>
-                    <Button sx={{backgroundColor:'#1F2937'}} variant="contained" size={"small"} onClick={handleClickOpen}>
+                    <Button sx={{backgroundColor: '#1F2937'}} variant="contained" size={"small"}
```

Comment:
Du könntest hier ain der gesamten Datei alle sx-Befehle löschen und folgendes in die css datei einfügen, leider geht das nur per !important,  weil ja MUI schon selbst designed und das überschreiben werden muss (die Farben sind nur platzhalter)
.css-1t1j96h-MuiPaper-root-MuiDialog-paper{
    background-color:green!important;
}
.css-1t1j96h-MuiPaper-root-MuiDialog-paper button{color:yellow;}
.css-11qr2p8-MuiButtonBase-root-MuiButton-root{background-color: red!important;}